README.md

```diff
@@ -198,6 +198,12 @@ Or run it without zookeeper container:
 
     docker run marathon:{version} --master local --zk zk://localhost:2181/marathon
 
+When using Docker on Mac you have to use a [special address](https://docs.docker.com/docker-for-mac/networking/#there-is-no-docker0-bridge-on-macos) instead of `localhost`:
+
+    docker run -p 8080:8080 --hostname $(hostname) -P mesosphere/marathon:v1.6.335 \
+        --master docker.for.mac.host.internal:505 \
```

Comment:
this is port `:5050`

however I don't think this completely works as mentioned in the previous comment.   Did you get it to work?